modules/Auth/lib/AuthHandler.cpp

```diff
@@ -98,6 +98,7 @@ TokenHandlingResult AuthHandler::handle_token(const ProvidedIdToken& provided_to
             req.id_tag.emplace(provided_token.id_token);
             this->stop_transaction_callback(this->connectors.at(connector_used_for_transaction)->evse_index, req);
```

Comment:
If I understand correctly this call can be used to trigger the "detach cable" - since we want to display the tick before that the publish call should happen before the stop call - same for the line 146

Reply:
Yes, of course it can. However, I can't imagine this happening without a state machine, so the order of the messages is not so important in that case.
If we will receive information that the Token is being processed, we can move to state which will show hourglass and wait only on three states to appear. 
The stop transaction will be stored in the queue until it can be processed in given state, so it does not matter if we have the accepted before that of after, right?

Reply:
While we may or may not have a state machine - is there any downside you see by publishing the messages in a "chronological" order?

Reply:
Not really, that does not concern me. I will change it